src/status_im/ui/screens/communities/community_overview_redesign.cljs

```diff
@@ -3,70 +3,137 @@
+             [status-im.ui.components.list.views :as list]
              [status-im.utils.handlers :refer [<sub >evt]]
              [status-im.ui.screens.communities.styles :as styles]
+             [quo2.foundations.colors :as colors]
              [status-im.ui.screens.communities.icon :as communities.icon]))
 
-(defn preview-list-component []
-  [react/view {:style {:height 20 :margin-top 20}}
+;; Mocked list items
+(def user-list
+  [{:full-name "Alicia K"}
+   {:full-name "Marcus C"}
+   {:full-name "MNO PQR"}
+   {:full-name "STU VWX"}])
+
+(defn preview-user-list []
+  [react/view {:style {:flex-direction :row
+                       :align-items :center
+                       :margin-top 20}}
+   [quo2.preview-list/preview-list {:type :user
+                                    :user user-list :list-size "4" :size 24}]
    [quo2.text/text {:accessibility-label :communities-screen-title
-                    :weight              :semi-bold
+                    :style {:margin-left 8}
                     :size                :label}
-    "placeholder for preview list component"]])
+    "Join Alicia, Marcus and 2 more"]]) ;; TODO remove mocked data and use from contacts list/communities members
+
+(def list-of-channels {:Welcome [{:name "welcome"
+                                  :emoji "🤝"}
+                                 {:name  "onboarding"
+                                  :emoji "🍑"}
+                                 {:name "intro"
+                                  :emoji "🦄"}]
+                       :General [{:name  "general"
+                                  :emoji "🐷"}
+                                 {:name  "people-ops"
+                                  :emoji "🌏"}
+                                 {:name "announcements"
+                                  :emoji "🎺"}]
+                       :Mobile [{:name "mobile"
+                                 :emoji "👽"}]})
 
 (defn channel-list-component []
-  [react/view {:style {:height 200 :margin-top 20}}
-   [quo2.text/text {:accessibility-label :communities-screen-title
-                    :weight              :semi-bold
-                    :size                :label}
-    "placeholder for channel list component"]])
+  [react/scroll-view {:style {:margin-top 20}}
+   [react/view {:style {:flex 1}}
+    (map (fn [category]
+           ^{:key (get category 0)}
+           [react/view {:flex 1}
+            [quo2.divider-label/divider-label {:label (get category 0)
+                                               :chevron-position :left}]
+            [react/view
+             {:margin-left 8
+              :margin-top 10
+              :margin-bottom 8}
+             [list/flat-list
+              {:shows-horizontal-scroll-indicator false
+               :separator [react/view {:margin-top 4}]
+               :data                              ((get category 0) list-of-channels)
+               :render-fn quo2.channel/list-item}]]]) list-of-channels)]])
```

Comment:
[Code style] I'm commenting on this particular line, but there are others in this PR. Not breaking arguments that span over multiple lines makes for a very difficult code to read in Lisps. For example, I couldn't figure out that `list-of-channels` belonged to the `map` call.

https://github.com/bbatsov/clojure-style-guide#function-arguments-alignment

According to the style guide it would look like this:

```clojure
(defn channel-list-component []
  [react/scroll-view {:style {:margin-top 20}}
   [react/view {:style {:flex 1}}
    (map (fn [category]
           ^{:key (get category 0)}
           [react/view {:flex 1}
            [quo2.divider-label/divider-label {:label            (get category 0)
                                               :chevron-position :left}]
            [react/view
             {:margin-left   8
              :margin-top    10
              :margin-bottom 8}
             [list/flat-list
              {:shows-horizontal-scroll-indicator false
               :separator                         [react/view {:margin-top 4}]
               :data                              ((get category 0) list-of-channels)
               :render-fn                         quo2.channel/list-item}]]])
         list-of-channels)]])
```